web/src/components/organisms/Table.jsx

```diff
@@ -31,7 +31,7 @@ const Table = ({ headers, data, showActions = true }) => {
                     </a>
                   </div>
                   <div className='text-blue-500 text-xl'>
-                    <a href='#'>
+                    <a href='order/{id}'>
```

Comment:
i think this is the error, should be row.id
- always use <Link> by nextjs 
- to concat variables should be {`text ${variableName}`}
```suggestion
                    <Link href={`order/${row.id}`}>
```